types/ldapjs/index.d.ts

```diff
@@ -75,8 +75,19 @@ export var Change: {
 	new(change: Change): Change;
 }
 
+export type SearchReference = any;
+
+export class SearchCallbackResponse extends EventEmitter {
+    on(event: string | symbol, listener: (...args: any[]) => void): this;
```

Comment:
Please move this to the bottom of the list of overloads for `on`; otherwise, this case will always be chosen.

Reply:
I don't know exactly how this works, that was my impression too before looking at [node typings](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fcf24503eb7bfebeb6279e2a5b341f078c73df87/types/node/v8/index.d.ts#L5115) which I finally used as reference.
Also tests work just fine matching with the correct overload.

Reply:
I just got a chance to ask around. I was incorrect in stating that this case will always be chosen.
It's true that for most overloads, the first will be chosen, but there's actually special handling for this particular case (i.e. string literals vs. just `string`) from before string literal types were widely usable to effectively "move the more specific ones to the top". As a result, the way the node typings are written are an artifact of that older handling. So this will work fine, but it's more consistent style with all overloads to move it to the bottom. Would you like to move the general ones to the bottom? If not, I'll go ahead and merge.

Reply:
Ok, I'm glad to do it if now is the more consistent way. I tried to be consistent with node typings :)

Reply:
Thank you!